docs/contribute/release_checklist.md

```diff
@@ -6,59 +6,43 @@ permalink: /contributing/release-checklist
+    }
 
+    /**
     ```
-    * [Release notes](https://mozac.org/changelog/#9800)
-    * [Commits](https://github.com/mozilla-mobile/android-components/compare/v97.0.0...v98.0.0)
-    * [Milestone](https://github.com/mozilla-mobile/android-components/milestone/145?closed=1)
-    ```
-
-    - Push the "Publish release.
-
-    <img width="1385" alt="Screen Shot 2022-02-08 at 9 55 58 AM" src="https://user-images.githubusercontent.com/1190888/153047976-cffdf1c1-e3c1-45eb-83ed-cedd021a68b4.png">
-
-5. After 30-60 minutes, follow up to see if a `[beta_version].0.0` build is available in one of the AC components on [maven/components](https://maven.mozilla.org/?prefix=maven2/org/mozilla/components/).
+3. Create a commit and pull request named `Switch to GeckoView Beta` for this change. The pull request should be committed into the `releases/[beta_version].0` branch. Land the pull request. This will create busted builds which are expected. The next step should green it up.
+4. Let the Github Action bump the geckoview version like in [#12485](https://github.com/mozilla-mobile/android-components/pull/12485).
+5. Once both pull requests are landed, go to [Shipit](https://shipit.mozilla-releng.net/) and kick off a new release.
+6. After 30-60 minutes, follow up to see if a `[beta_version].0.0` build is available in one of the AC components on [maven/components](https://maven.mozilla.org/?prefix=maven2/org/mozilla/components/).
+7. Tell the dev team they can handle the changes down below ⬇️
 
-See [https://github.com/mozilla-mobile/android-components/pull/11520](https://github.com/mozilla-mobile/android-components/pull/11520) for an example.
+## [Dev team] Starting the next Nightly development cycle
 
-## Starting the next Nightly development cycle
+**Please handle this part once Release Management gives you the go.**
 
+0. Wait for greenlight coming from Release Management.
 1. Create a local development branch off of the `main` branch.
-2. Create a commit named `Start [nightly_version].0.0 development cycle` In [.buildconfig.yml](https://github.com/mozilla-mobile/android-components/blob/main/.buildconfig.yml) and [version.txt](https://github.com/mozilla-mobile/android-components/blob/main/version.txt), bump the current `componentsVersion` up by 1 and ensure this change is also synced with `version.txt`. `[nightly_version]` should follow the Firefox Nightly release number. See [Firefox Release Calendar](https://wiki.mozilla.org/Release_Management/Calendar).
+2. Create a commit named `Start [nightly_version].0.0 development cycle` in [version.txt](https://github.com/mozilla-mobile/android-components/blob/main/version.txt), bump the current `componentsVersion` up by 1 and ensure this change is also synced with `version.txt`. `[nightly_version]` should follow the Firefox Nightly release number. See [Firefox Release Calendar](https://wiki.mozilla.org/Release_Management/Calendar).
```

Comment:
This ^^ and the code snippet below are still referring to `componentsVersion` in `.buildconfig.yml` which doesn't exist anymore. Just need to update `version.txt` now, right?

Reply:
Oops, you're totally right! Fixed.